.github/workflows/qodana_code_quality.yml

```
@@ -0,0 +1,53 @@
+        id: setup-php
+        if: steps.checkout.outcome == 'success'
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "8.2"
+          extensions: ast, mbstring, pdo, pdo_mysql, xml, zip
+          coverage: xdebug
+
+      - name: Cache Composer Dependencies
+        id: composer-cache
+        if: steps.setup-php.outcome == 'success'
+        uses: actions/cache@v2
```

actionlint (1.7.7)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)